pkg/config/remote/client/client.go

```diff
@@ -589,19 +600,19 @@ func (c *Client) newUpdateRequest() (*pbgo.ClientGetConfigsRequest, error) {
 		CachedTargetFiles: pbCachedFiles,
 	}
 
-	switch c.Options.isUpdater {
-	case true:
```

Comment:
No need to go so far as to create Go's excuse for an enum (*forever sad about this*), but I do think this is a good use case for at least a constant string.

Reply:
force pushed with consts added 👍 good call https://github.com/DataDog/datadog-agent/pull/29779/files#diff-ac7ce945899bf069e091710895b0a5bc7376daf4053bcae1f50cae8ce1285a77R43-R45